### PR TITLE
Aw 80 quiz

### PR DESCRIPTION
**Pull request by Pacane**
- Had to turn on auto-convert of css into gss because of gwtQueryTooltip plugin.
- There "Share" function of the quiz is still missing.
- The tooltips still has to be styled
- Gotta get rid of Christian's face

### CODE REVIEW

File: src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionPresenter.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        // TODO: Implement share
+    }
+
+    @Override
+    protected void onBind() {
+        getView().setQuestion(1);
+    }
+
+    @Inject
+    QuestionPresenter(
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionPresenter.java]: I think ctor goes on top

File: src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    public void onAttachOrDetach(AttachEvent attachEvent) {
+        if (attachEvent.isAttached()) {
+            createTooltips();
+        } else {
+            destroyTooltips();
+        }
+    }
+
+    private void destroyTooltips() {
+        $("[title]").as(Tooltip).destroy();
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.java]: You should limit the scope of the selector, maybe `$("[title]", asWidget())`

File: src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    }
+
+    private void createTooltips() {
+        TooltipOptions options = new TooltipOptions()
+                .withDelayHide(100)
+                .withDelayShow(200)
+                .withPlacement(TooltipOptions.TooltipPlacement.RIGHT);
+        $("[title]").as(Tooltip).tooltip(options);
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.java]: Same comment

File: src/main/java/com/arcbees/website/client/resources/PageBeesResources.java [hunk 4]
```diff
@@ -89,4 +90,14 @@
...
+    interface QuizMessages extends Messages {
+        String question(@PluralCount int questionNumber);
+
+        String answer1(@PluralCount int questionNumber);
+
+        String answer2(@PluralCount int questionNumber);
+
+        String answer3(@PluralCount int questionNumber);
+    }
+}
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/resources/PageBeesResources.java]: nl at eof

File: src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties [hunk 5]
```diff
@@ -799,4 +799,33 @@ D326FE28E7235EB8CCF4507BB09F0394=GWTP exploite la force de la génération de co
...
+answer1[\=1]=Code, Code, Code
+answer2[\=1]=Prendre une bière avec mes collègues
+answer3[\=1]=J''essaie de contacter E.T. et de me faire de nouveaux amis
+
+answer1[\=2]=vraiment mauvais. J''ai deux mains gauches.
+answer2[\=2]=bon, mais je peux m''améliorer.
+answer3[\=2]=le champion incontesté!
+
+answer1[\=3]=C''est quoi Google? Moi j''utilise Bing.
+answer2[\=3]=J''aime Google autant que j'aime les bagels.
```
Comment (meriouma) [inline src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties]: double quote

File: src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties [hunk 6]
```diff
@@ -799,4 +799,33 @@ D326FE28E7235EB8CCF4507BB09F0394=GWTP exploite la force de la génération de co
...
+answer3[\=3]=Vers l''infini et plus loin encore!
+
+answer1[\=4]=Facebook, quoi d'autres?
```
Comment (meriouma) [inline src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties]: double quote

File: src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.ui.xml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+                <g:Image ui:field="answer3Image" resource="{page.beeChristian}"/>
+            </label>
+            <input type="radio" name="quiz" id="answer3" value="answer3"/>
+            <g:Button ui:field="nextButton" styleName="{res.style.btn}">Next</g:Button>
+        </div>
+        <div ui:field="quizFinished" class="{res.style.hidden}">
+            <!--TODO: Add quiz finished content -->
+            Quiz finished!
+            <g:Button ui:field="share" styleName="{res.style.btn}">Share</g:Button>
+            <g:Button ui:field="tryAgain" styleName="{res.style.btn}">Try again</g:Button>
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.ui.xml]: Should those texts be `ui:msg`?

File: src/main/java/com/arcbees/website/client/application/bees/BeesView.java [hunk 8]
```diff
@@ -110,4 +120,15 @@ public void f() {
     public void konami() {
         $("img", bees).attr("src", "http://placekitten.com/200/300");
     }
+
+    @Override
+    public void initQuiz() {
+        $(initQuizButton).hide();
+
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/bees/BeesView.java]: rm nl

**Comment by meriouma** [comment]
LGTM

File: src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.ui.xml (hunk 7 above)
Comment (Pacane) [inline src/main/java/com/arcbees/website/client/application/bees/quiz/QuestionView.ui.xml]: Yes probably.

**Comment by jasonlemay** [comment]
LGTM

**Pull request merged**